src/views/annual-report/2020/annual-report.jsx

```diff
@@ -0,0 +1,2367 @@
+const bindAll = require('lodash.bindall');
+const classNames = require('classnames');
+const React = require('react');
+const MediaQuery = require('react-responsive').default;
+const FormattedMessage = require('react-intl').FormattedMessage;
+const injectIntl = require('react-intl').injectIntl;
+const intlShape = require('react-intl').intlShape;
+
+const render = require('../../../lib/render.jsx');
+const frameless = require('../../../lib/frameless');
+
+const Avatar = require('../../../components/avatar/avatar.jsx');
+const Page = require('../../../components/page/www/page.jsx');
+const Grid = require('../../../components/grid/grid.jsx');
+const Button = require('../../../components/forms/button.jsx');
+const FlexRow = require('../../../components/flex-row/flex-row.jsx');
+const Comment = require('../../../components/comment/comment.jsx');
+const CountryBlurb = require('../../../components/country-blurb/country-blurb.jsx');
+const TextAndMediaSnippet = require('../../../components/text-and-media-snippet/text-and-media-snippet.jsx');
+const TimelineCard = require('../../../components/timeline-card/timeline-card.jsx');
+const WorldMap = require('../../../components/world-map/world-map.jsx');
+const CountryUsage = require('./country-usage.json');
+const IndiaProjects = require('./india-projects.json');
+const PeopleGrid = require('../../../components/people-grid/people-grid.jsx');
+const People = require('./people.json');
+const VideoPreview = require('../../../components/video-preview/video-preview.jsx');
+const Supporters = require('./supporters.json');
+import {TwitterTweetEmbed} from 'react-twitter-embed';
+
+
+require('./annual-report.scss');
+
+// Founder’s Message / Mission / Reach / Themes / Director’s Message / Supporters / Team / Donate
+
+// Some constants used for the page subnav and section refs
+const SECTIONS = {
+    founders_message: 'founders-message',
```

Comment:
One of these strings has a hyphen and the other has an underscore... which is different from directors message below:
```suggestion
    founders_message: 'founders_message',
```